crates/ethereum/payload/src/config.rs

```diff
@@ -19,4 +22,13 @@ impl EthereumBuilderConfig {
     pub fn extra_data(&self) -> Bytes {
         self.extra_data.clone()
     }
+
+    /// Returns the gas limit for the next block based
+    /// on parent and desired gas limits.
+    pub fn gas_limit(&self, parent_gas_limit: u64) -> u64 {
+        let delta = parent_gas_limit / GAS_LIMIT_BOUND_DIVISOR;
+        let min_gas_limit = parent_gas_limit + delta - 1;
+        let max_gas_limit = parent_gas_limit - delta + 1;
+        parent_gas_limit.clamp(min_gas_limit, max_gas_limit)
+    }
```

Comment:
we need a standalone function that accepts two u64, that this fn then calls

should be upstreamed to alloy once merged